src/tutorials/0005-regular-files-api/03.md

```diff
@@ -8,7 +8,7 @@ Here in our ProtoSchool tutorials, we create a new IPFS node for you in the brow
 
 We create these IPFS nodes behind the scenes so that you can focus on the content of each lesson.  Outside of ProtoSchool, though, you'd want a more consistent experience and a node (or multiple nodes) that you could access repeatedly. To achieve this, you could either initialize [JS IPFS in the browser](https://github.com/ipfs/js-ipfs#use-in-the-browser) on your own, or host your own node locally by installing IPFS and running a daemon in your terminal. When you're ready to experiment with that, you can visit the IPFS docs site to learn how to [install IPFS](https://docs.ipfs.io/guides/guides/install/) and [initialize your node](https://docs.ipfs.io/introduction/usage/#initialize-the-repository).
```

Comment:
This link seems to also need an update: https://github.com/ipfs/js-ipfs#use-in-the-browser

Maybe we could replace with one of these:

- https://github.com/ipfs/js-ipfs/tree/master/examples
- https://github.com/ipfs/js-ipfs/tree/master/examples/browser-script-tag

Reply:
Nice catch. Switching to this: https://github.com/ipfs/js-ipfs/blob/master/docs/BROWSERS.md